### ISSUE TITLE
[Anon] Bug report: T'au Empire.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#5974]
**File:** T'au Empire.catz

**BattleScribe version:** 2.03.11

**Platform:** Android

**Dropbox:** No

**Description:** Hello. I very much enjoy the app. However, lately, I have encountered two errors, both when building lists with the T'au Empire. The first is this: I am able to give an XV104 Riptide battlesuit a Shield generator when that should never be possible according to the rules. 

Additionally, I have attempted to take a second relic with my army, which is legal assuming you select emergency dispensation. However, whenever I choose a second relic, I am bombarded with error messages that are seemingly without end. I hope my report was detailed enough to assist, and thank you for your time.

**Comment by Mad-Spy** [comment]
I'm unable to replicate the problem with the Riptide in the most recent version of the app (2.03.12) with the latest data (T'au v49).

The other issue is caused by a Battlescribe app bug which we're waiting on a fix for.

**Comment by amis92** [comment]
Since the first is resolved and the second is a duplicate of #5955, I'll close this one.
